fhi_users/auth/rest_auth_views.py

```
@@ -371,15 +372,23 @@ def perform_create(
-                UserDomain.find_by_name(name=domain_name).get()
+                if not domain_name or len(domain_name) == 0:
+                    raise UserDomain.DoesNotExist()
+                user_domain_opt = UserDomain.find_by_name(name=domain_name).get()
             except UserDomain.DoesNotExist:
                 try:
-                    UserDomain.objects.get(visible_phone_number=visible_phone_number)
+                    user_domain_opt = UserDomain.objects.get(
+                        visible_phone_number=visible_phone_number
+                    )
                 except:
+                    logger.opt(exception=e).error(
```

Ruff (0.8.2)

Undefined name `e`

(F821)